.github/actions/build_test_artifacts/action.yml

```diff
@@ -0,0 +1,24 @@
+name: Build Test Artifacts
```

Comment:
On Terra, we have `Build Test Contracts` while here it's called `Build Test Artifacts`. Let's try to keep more consistency between the two.

Is this only for testing, or could we reuse the action for `.github/workflows/contracts-release.yml` workflow?